tests/vitest/io/export.test.ts

```
@@ -518,145 +419,333 @@ describe(`Export functionality comprehensive tests`, () => {
+      globalThis.URL.createObjectURL = vi.fn(() => mock_url)
+      globalThis.URL.revokeObjectURL = vi.fn()
+      globalThis.Image = vi.fn(() => mock_image) as unknown as typeof Image
+    })
+
+    it(`exports PNG with correct dimensions and DPI`, () => {
+      export_svg_as_png(mock_svg, `f.png`, 150)
+      expect(mock_canvas.width).toBe(208)
+      expect(mock_canvas.height).toBe(208)
+      expect(mock_image.crossOrigin).toBe(`anonymous`)
+      expect(mock_image.src).toBe(mock_url)
+      if (typeof mock_image.onload === `function`) {
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)